lib/pbench/server/api/auth.py

```diff
@@ -1,3 +1,4 @@
+from http import HTTPStatus
```

Comment:
This line should be positioned with the other `from` imports, in alphabetical order.

And, while you're at it, could you order the bare `import`'s, and add blank lines between the bare `import`'s, the `from` `import`'s, and the project `import`'s ?